docs/en/command-events.md

```diff
@@ -0,0 +1,142 @@
+|                                                       |       | Yiisoft\Yii\Debug\Collector\Console\ConsoleAppInfoCollector::collect |
+| Yiisoft\Yii\Console\Event\ApplicationShutdown         | 2     | Yiisoft\Yii\Debug\Collector\Console\ConsoleAppInfoCollector::collect |
+|                                                       |       | Yiisoft\Yii\Debug\Debugger::shutdown                                 |
+| Symfony\Component\Console\Event\ConsoleCommandEvent   | 2     | Yiisoft\Yii\Debug\Collector\Console\ConsoleAppInfoCollector::collect |
+|                                                       |       | Yiisoft\Yii\Debug\Collector\Console\CommandCollector::collect        |
+| Symfony\Component\Console\Event\ConsoleErrorEvent     | 3     | Yiisoft\Yii\Debug\Collector\Console\ConsoleAppInfoCollector::collect |
+|                                                       |       | Yiisoft\Yii\Debug\Collector\Console\CommandCollector::collect        |
+|                                                       |       | Yiisoft\Yii\Console\ErrorListener::onError                           |
+| Symfony\Component\Console\Event\ConsoleTerminateEvent | 3     | Yiisoft\Yii\Debug\Collector\Console\ConsoleAppInfoCollector::collect |
+|                                                       |       | Yiisoft\Yii\Debug\Collector\Console\CommandCollector::collect        |
+|                                                       |       | static function (\Psr\Log\LoggerInterface $logger): void {           |
+|                                                       |       |     if ($logger instanceof \Yiisoft\Log\Logger) {                    |
+|                                                       |       |         $logger->flush(true);                                        |
+|                                                       |       |     }                                                                |
+|                                                       |       | }                                                                    |
+| Yiisoft\Yii\Cycle\Event\AfterMigrate                  | 1     | Yiisoft\Yii\Cycle\Listener\MigrationListener::onAfterMigrate         |
++-------------------------------------------------------+-------+----------------------------------------------------------------------+
+```
+### List all groups
+
+You can list all groups by specifying `--groups` option:
+
+```
+❯ ./yii debug:events --groups
+ ---------------------- 
+  Groups                 
+ ---------------------- 
+  params-console        
+  bootstrap-console     
+  di-console            
+  di-providers-console  
+  di-delegates-console  
+  events-console        
+  widgets               
+  widgets-themes        
+ ---------------------- 
+```
+
+> Note: unfortunately, the command cannot recognize only event groups automatically, so you will have all groups listed.
```

Comment:
What does it mean?

Reply:
If you run `debug:events --groups` you can see all configuration groups like `params`, `di`, `bootstrap`, `events` etc. 

The only `events` is currently contains event subscribers. But in a user space it may be named as they want. That's why `--groups` lists all config groups.

Reply:
OK.